kinds/dashboard/dashboard_kind.cue

```diff
@@ -85,10 +85,18 @@ lineage: seqs: [
 				templating?: {
 					list?: [...#VariableModel] @grafanamaturity(NeedsExpertReview)
 				}
-				// TODO docs
-				annotations?: {
+
+				// TODO -- should not be a public interface on its own, but required for Veneer
+				#AnnotationContainer: {
```

Comment:
@sdboyer -- thoughts on how to best handle this?  a nested type `AnnotationQuery.target` needs a Veneer... but that sets off a cascade of all the parents needing one also

Reply:
the cascade being this change, for example? https://github.com/grafana/grafana/pull/66325/files#diff-932ad9420b4f3774837637528ee5bfe0ac3581cca3c5fdad61beb944aedbc45aR33

nope, no current or planned way around that. Would require the sort of generics support in cuetsy that i haven't yet seen what i consider to be a feasible proposal

Reply:
not a huge deal -- just checking that I am not missing anything.